zds/member/views.py

```diff
@@ -956,13 +961,13 @@ def get_client_ip(request):
         # other
         return request.META.get('REMOTE_ADDR')
     else:
-        # should never happend
+        # Should never happen.
         return '0.0.0.0'
 
 
 @login_required
 def settings_promote(request, user_pk):
-    """ Manage the admin right of user. Only super user can access """
+    """ Manage the admin right of user. Only super user can access."""
```

Comment:
La phrase ne correspond pas à ce que fait la vue (et ce n'est pas ta faute). Je propose ceci : "Manages groups and activation status of a user. Only superusers are allowed to use this.".